dev-tools/tarball-compare/Main.hs

```diff
@@ -118,30 +119,40 @@ main =
                     entries <- Tar.pack "." $ defFile : files
                     -- need to force the tar entries, withCurrentDirectory is not retained
                     mapM_ (`seq` pure ()) entries
-                    pure $ foldr Tar.Next Tar.Done entries
+                    error "Unimplemented: can't make it typecheck!!" -- pure $ foldr Tar.Next Tar.Done entries
```

Comment:
...maybe we should consider removing this tool?
It relies upon the directory names and request IDs in the tar file being equal for the same proof - which is not true any more since recent changes were made (to both directory names and request IDs).
I haven't used the tool in quite a while, it was for conformance checks between `booster` and `kore-rpc`.

Reply:
Let's delete, can always resurrect if needed!